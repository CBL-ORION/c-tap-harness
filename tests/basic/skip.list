skip
skip-all
skip-all-quiet
pass
order
