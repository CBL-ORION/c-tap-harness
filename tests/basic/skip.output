
Running all tests listed in skip.list.  If any tests fail, run the failing
test program with runtests -o to see more details.

skip............ok (skipped 5 tests)
skip-all........skipped (some reason)
skip-all-case...skipped (some reason)
skip-all-quiet..skipped
pass............ok
order...........ok

All tests successful, 8 tests skipped.
Files=6,  Tests=18
