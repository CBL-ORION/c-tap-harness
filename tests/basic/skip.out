
Running all tests listed in skip.list.  If any tests fail, run the failing
test program by hand to see more details.  The test program will have the
same name as the test set but with "-t" or ".t" appended.

skip............ok (skipped 3 tests)
skip-all........skipped
	all skipped: some reason
skip-all-quiet..skipped
pass............ok
order...........ok

All tests successful, 5 tests skipped.
Files=5,  Tests=20
